packages/libs/pactjs-test-project/src/example-templates/generated-tx-templates/index.ts

```diff
@@ -0,0 +1,115 @@
+// generated by pactjs-generator and pactjs-cli@0.0.4
+import { buildUnsignedTransaction, ICommandBuilder } from '@kadena/client';
+
+export default {
+  "safe-transfer": (args: {
+    "fromAcct": string,
+    "toAcct": string,
+    "amount": string,
+    "chain": string,
+    "network": string,
+    "fromKey": string,
+  }): ICommandBuilder<{}> => {
+    const parts = [
+      "code: |-\n  (coin.transfer \"",
+      "\" \"",
+      "\" ",
+      ")\ndata:\npublicMeta:\n  chainId: \"",
+      "\"\n  sender: ",
+      "\n  gasLimit: 2500\n  gasPrice: 1.0e-8\n  ttl: 600\nnetworkId: ",
+      "\nsigners:\n  - pubKey: ",
+      "\n    caps:\n      - name: \"coin.TRANSFER\"\n        args: [",
+      ", ",
+      ", ",
+      "]\n      - name: \"coin.GAS\"\n        args: []\ntype: exec\n"
+    ];
+    const holes = [
+      "fromAcct",
+      "toAcct",
+      "amount",
+      "chain",
+      "fromAcct",
+      "network",
+      "fromKey",
+      "fromAcct",
+      "toAcct",
+      "amount"
+    ];
+
+    return buildUnsignedTransaction(parts, holes, args);
+  },
+  
+  "transfer.json": (args: {
+    "fromAcct": string,
+    "toAcct": string,
+    "amount": string,
+    "chain": string,
+    "network": string,
+    "fromKey": string,
+  }): ICommandBuilder<{}> => {
+    const parts = [
+      "{\n  \"code\": \"(coin.transfer \\\"",
```

Comment:
Would it make sense to rewrite the string to a template string? So we could omit the escaping `\`?